static/app/views/settings/project/filtersAndSampling/modals/errorRuleModal.tsx

```diff
@@ -26,15 +26,15 @@ class ErrorRuleModal extends Form<Props, State> {
     const {rule} = this.props;
 
     if (rule) {
-      return t('Edit a custom rule for errors');
+      return t('Edit Error Sampling Rule');
     }
 
-    return t('Add a custom rule for errors');
+    return t('Add Error Sampling Rule');
   }
 
   geTransactionFieldDescription() {
     return {
-      label: t('Error'),
+      label: t('Errors'),
       // help: t('This is a description'), TODO(PRISCILA): Add correct description
```

Comment:
@doralchan Please, do you have some idea of what I can add here? The descriptions were not yet provided

Reply:
Sure - I'll add it to transactions too

Reply:
nice! Thanks 🙏